core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -638,6 +660,11 @@ public interface MetricsProducer {
   String METRICS_TSERVER_SCAN_RESULTS_BYTES = METRICS_TSERVER_PREFIX + "scan.results.bytes";
   String METRICS_TSERVER_SCANNED_ENTRIES = METRICS_TSERVER_PREFIX + "scan.scanned.entries";
 
+  String METRICS_SSERVER_PREFIX = "accumulo.sserver.";
+  String METRICS_SSERVER_REGISTRATION_TIMER = METRICS_SSERVER_PREFIX + "registration.timer";
+  String METRICS_SSERVER_BUSY_COUNTER = METRICS_SSERVER_PREFIX + "busy.count";
+  String METRICS_SSERVER_TABLET_METADATA_CACHE = METRICS_SSERVER_PREFIX + "tablet.metadata.cache";
```

Comment:
Wondering if [this](https://github.com/apache/accumulo/blob/6c8e175eab871428e0340d65bdcedbef0d332c42/core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java#L608) should be `accumulo.scans` instead of `accumulo.tserver.scans`.  That may be a change for main instead and not 2.1.3 though.  As you mentioned the tag will indicate what kind of process it is.

Reply:
Changed in 5dd3c2d311

Reply:
Addition clean-up to remove tserver from scan metrics in a8235a8.  Also added query to tag where needed so that the tags are not `scan.scan`